internal/servers/controller/testing.go

```diff
@@ -424,7 +424,6 @@ func NewTestController(t *testing.T, opts *TestControllerOpts) *TestController {
 		cancel: cancel,
 		opts:   opts,
 	}
-
```

Comment:
Unless this is added through `make fmt` you should avoid making these types of changes as it pollutes the files history and git blame.

Reply:
I was using the merge feature of my IDE and have no idea what is going on here